frontend/src/queries/nodes/DataTable/queryFeatures.ts

```diff
@@ -59,5 +60,9 @@ export function getQueryFeatures(query: Node): Set<QueryFeature> {
         features.add(QueryFeature.resultIsArrayOfArrays)
     }
 
+    if (isWebStatsTableQuery(query)) {
+        features.add(QueryFeature.hideFooterIfCantLoadMore)
+    }
+
```

Comment:
This seems like a very very very specific feature 🤔

Reply:
> This seems like a very very very specific feature 🤔

![untitled(1)](https://github.com/PostHog/posthog/assets/2056078/157ad7f1-9d72-4453-916d-29d1922d124d)

It's to hide this^ which is imo quite confusing atm. I think there's been a few situations like, where I want to make slightly different UI decisions in web analytics than were made in insights, but don't want to do a full rewrite. E.g. adding VizSpecificOptions here https://github.com/PostHog/posthog/blob/f0c4908aba87f1713d305ed50f4d5165f4784837/frontend/src/queries/schema.ts#L467

Reply:
We should definitely change the word "events" to "entries" in that message. Feel free to do here.

Reply:
> I wonder why you didn't just make the button a link to urls.insightNew(..., query) that just opens in a new window? It's equally full screen and allows you to dive even deeper into the data if you want to.

There's a few kinks to iron out before WebStatsTableQuery nodes can be opened as a new insight. I'm closing this PR in favour of https://github.com/PostHog/posthog/pull/20148